packages/postcss-config/lib/index.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * WordPress dependencies
+ */
+const { adminColorSchemes } = require( '@wordpress/base-styles' );
+
+module.exports = {
+	plugins: () => [
```

Comment:
Should this be a function? Or just the array? I guess it's convenient for how we configure Webpack currently, since it's assigning a function for `postcss-loader` configuration.

But looking at [the options for the loader](https://github.com/postcss/postcss-loader#options), it seems that it should also accept as an array? And, separately, this isn't just useful for Webpack, so we shouldn't be overly concerned with targeting Webpack specifically.

I guess it could still make sense as a function if we'd need to accept options at some point in the future, and generate an array of plugins based on those options.

Reply:
It turned out that PostCSS config is a thing as explained in https://github.com/WordPress/gutenberg/pull/21730#issuecomment-621072189. I need to explore it further but it feels like using an array for plugins is all we need to make it work everywhere and follow the guidelines for proper config file.

Reply:
I renamed the package to `@wordpress/postcss-plugins-preset` and it returns an array (c256fc006845815231289f3d42afe12855ccbc4d).